src/components/content/catalog/services/ServiceProvider.tsx

```diff
@@ -109,12 +111,44 @@ function ServiceProvider({
         setActiveKey(key);
     };
 
+    function setUnregisterTipsInfo(type: 'error' | 'success', msg: string) {
+        setUnregisterTips(
+            <div className={'submit-alert-tip'}>
+                {' '}
+                <Alert
+                    message='Unregister:'
+                    description={msg}
+                    showIcon
+                    type={type}
+                    closable={true}
+                    onClose={onRemove}
+                />{' '}
+            </div>
+        );
+    }
+
+    const onRemove = () => {
+        window.location.reload();
+    };
+
+    const [unregisterTips, setUnregisterTips] = useState<JSX.Element | undefined>(undefined);
+
+    const onConfirmUnregister = useCallback((type: string, msg: string) => {
```

Comment:
Do we really need to use the useCallback hook here? because the page will re-renders, it will anyways rerender UnregisterService component too.

Reply:
fixed.